setup.py

```diff
@@ -39,7 +39,7 @@ def build_media_pattern(base_folder, file_extension):
 
 setup(
     name = "django-wiki",
-    version="0.0.23",
+    version="0.0.24",
```

Comment:
You can hold off on the version increase for now, there are several more changes needed before this package will be ready to run under Django 2.2.